website/content/api-docs/jobs.mdx

```diff
@@ -2496,3 +2496,167 @@ $ curl \
+
+The table below shows this endpoint's support for
+[blocking queries](/nomad/api-docs#blocking-queries) and
+[required ACLs](/nomad/api-docs#acls).
+
+| Blocking Queries | ACL Required         |
+| ---------------- | -------------------- |
+| `NO`             | `namespace:read-job` |
+
+### Parameters
+
+- `:job_id` `(string: <required>)` - Specifies the ID of the job (as specified in
+  the job file during submission). This is specified as part of the path.
+
+### Sample Request
+
+```shell-session
+$ curl \
+    https://localhost:4646/v1/job/my-job/actions
+```
+
+### Sample Response
+
+```json
+[
+  {
+    "Args": [
+      "-s",
+      "wttr.in/Toronto?format=3"
+    ],
+    "Command": "/usr/bin/curl",
+    "Name": "weather",
+    "TaskGroupName": "group1",
+    "TaskName": "task"
+  },
+  {
+    "Args": [
+      "-c",
+      "nomad alloc status ${NOMAD_ALLOC_ID}"
+    ],
+    "Command": "/bin/sh",
+    "Name": "get alloc info",
+    "TaskGroupName": "group2",
+    "TaskName": "task"
+  },
+]
+```
+
+## Run Action
+
+This endpoint executes a predefined action from a job specification in the
+context of a specific task within an allocation. It opens a WebSocket to
+transmit output from the running action.
+
+
+| Method      | Path                     | Produces               |
+| ----------- | ------------------------ | ---------------------- |
+| `WebSocket` | `/v1/job/:job_id/action` | WebSocket JSON streams |
+
+The table below shows this endpoint's support for
+[blocking queries](/nomad/api-docs#blocking-queries) and
+[required ACLs](/nomad/api-docs#acls).
+
+| Blocking Queries | ACL Required                                                                                 |
+| ---------------- | -------------------------------------------------------------------------------------------- |
+| `NO`             | `namespace:alloc-exec` (and `namespace:alloc-node-exec` if target task uses raw_exec driver) |
+
+### Parameters
+
+- `:job_id` `(string: <required>)` - Specifies the ID of the job containing the
+  action. This is specified as part of the path.
+- `action` `(string: <required>)` - Specifies the name of the action to be
+  executed, as defined within the job's task. Specified as a query parameter.
+- `allocID` `(string: <required>)` - Specifies the UUID of the target
+  allocation. Specified as a query parameter.
+- `task` `(string: <required>)` - Specifies the task within the group where the
+  action is defined. Specified as a query parameter.
+- `tty` `(bool: false)` - Specifies whether a TTY is allocated for this action.
+  Specified as a query parameter.
+- `ws_handshake` `(bool: false)` - Specifies whether to expect the
+  authentication token in the first frame, as a query parameter.
+
+
+### WebSocket Upgrade Request
+
+To execute an action, a WebSocket upgrade request is sent. The request headers
+must include the standard WebSocket upgrade parameters. Example:
+
+```shell-session
+$ curl \
```

Comment:
As mentioned above, we're trying to move examples to `nomad operator api`, but it doesn't support `-o` 🤔 

In this example I think it would fine to omit `tty` and just output to stdout, but when TTY is required, I'm not sure how that would work. Maybe a redirect would do the trick? `nomad operator api ... > /dev/tty`

Nevertheless, I think it's fine to use `curl` here because of the extra functionality not supported by `nomad operator api`.